weaviate_cli/managers/data_manager.py

```diff
@@ -401,6 +406,7 @@ def __delete_data(
 
         for obj in data_objects:
             collection.with_consistency_level(cl).data.delete_by_id(uuid=obj.uuid)
+            print(f"Object deleted: {obj.uuid} into class '{collection.name}'")
```

Comment:
I wouldn't add this log, if we are deleting 1000 or 5000 objects it will throw this log 5000 times...

Reply:
Removed